main.js

```diff
@@ -0,0 +1,23 @@
+const arrows = document.getElementById('arrows');
+const container = document.getElementById('container');
+const rightArrow = document.getElementById('right');
+const leftArrow = document.getElementById('left');
+
+function moveImg(event) {
+    const idButton = event.target.id;

```

Comment:
ok, but for my mind will be better to use data attribute or classList.has. Also, you can use classes for styling